--- a/bin/regulartesting.pl
+++ b/bin/regulartesting.pl
@@ -16,9 +16,9 @@
 use perfSONAR_PS::RegularTesting::Master;
 use perfSONAR_PS::RegularTesting::Utils::ConfigFile qw(parse_file);
 
-my $CONFIG_FILE = "$RealBin/../etc/regulartesting.conf";
+my $CONFIG_FILE = "/etc/perfsonar/regulartesting.conf";
 my $PID_DIR  = "/var/run";
 my $PID_FILE = "regulartesting.pid";
 my $LOGGER_CONF;
 my $DEBUGFLAG;
 my $DAEMONIZE;
